### ISSUE TITLE
Moving the repository to libuv/libuv

### ISSUE THREAD

**Issue by saghul** [joyent/libuv#1573]
I’m posting this here, on behalf of the libuv core team, since it may have more reach than sending an email to the mailing list.

The 1.0.0 release is due soon, and after it’s done the development of libuv will be moved to its own organization (libuv), which currently just contains a fork of it. We kindly ask Joyent to transfer the project to this new organization.

We’ll take the opportunity to close lots of issues that have been lingering around for a long time and no longer apply, a fresh start :-) The issues that apply will be reopened in the new repository and unmerged pull requests will be dealt with in joyent/libuv until merged in libuv/libuv.

Why this? While libuv was started as part of the efforts to bring Node to Windows, and thus part of joyent/*, libuv has been “on its own” for quite a while now. Moreover, libuv has seen significant use outside of Node, which has made it less Node-centric. Joyent is not acting as the project steward, which is perfectly fine, since the community effectively took over the project: gave it a logo, (some) documentation and even changed the versioning scheme, it was only a matter of time that libuv wanted to fly solo.

This changes nothing with regards to the relationship between libuv and Joyent and / or Node; we won’t break Node just because, and I’ll personally continue to help adapt Node to changes we might introduce in further releases.

In a nutshell, business as usual, with a new location for the libuv repository.

**Comment by isaacs** [comment]
:clap: :+1:

**Comment by travisjeffery** [comment]
yesss :+1:

**Comment by mikeal** [comment]
:clap:

**Comment by interlock** [comment]
:clap:

**Comment by mikermcneil** [comment]
Cool :)

**Comment by adohe-zz** [comment]
cool :+1:

**Comment by saghul** [comment]
The move has happened. We are now starting fresh at https://github.com/libuv/libuv and some new ideas are already being proposed: https://github.com/libuv/libuv/labels/rfc

I'd still like to get the repo handed over though.

**Comment by Fishrock123** [comment]
:+1:

**Comment by lloydwatkin** [comment]
```
since the community effectively took over the project: gave it a logo, 
(some) documentation and even changed the versioning scheme, 
it was only a matter of time that libuv wanted to fly solo.
```

You forgot about becoming a t-shirt company too :)

Good luck!

**Comment by fengmk2** [comment]
:+1:

**Comment by recp** [comment]
:+1:

**Comment by kaushal** [comment]
:+1:

**Comment by joaojeronimo** [comment]
So... when is the same going to happen with node.js ;) ?

**Comment by gkze** [comment]
^ ^ :+1: :+1:

**Comment by Fishrock123** [comment]
@joaojeronimo Joyent has trademarks on "node.js". Also, the contributors and such are very different. it's a much more complex situation, unfortunately.

(note: this _is_ off-topic here. See https://github.com/node-forward)

**Comment by ghmlee** [comment]
Cool. :+1:

**Comment by pradeep-mishra** [comment]
ok :+1:

**Comment by tkelman** [comment]
Would be ideal to see this as a repository ownership move (so url's get automatically redirected, etc) instead of a fork. Hopefully that's planned but just hasn't happened yet.

**Comment by jmcbee** [comment]
:clap:

**Comment by Tushkiz** [comment]
Cool, keep up the good work. :clap:

**Comment by sjonnet19** [comment]
I would have proposed to move the repo as a such to keep issues, stars and followers.

**Comment by saghul** [comment]
I did ask in the original message, but its up to Joyent. It could still
happen, but if it doesn't I personally don't mind. Onwards and upwards!

**Comment by mcanthony** [comment]
Being that libuv competes with some well established libraries of a similar nature I am inclined to think Joyent may try to hold onto libuv for the "innovation" it represents. I assume because of the way this thread reads no one had contacted Joyent prior to the decision to declare the project community ran (officially and otherwise). If Joyent had been approached by the community this move would not look like a hostile takeover, forced by way of a trivial twitter/comment induced campaign to alter the outside communities perception of where the project stands.

That being said I would certainly welcome a fork of the libuv project that brings with it more ideas and better maintenance, however I am wary about the way it seems to have been imposed (this is my uninformed assumption) and I have some concerns about an identically named fork if Joyent was to decide not to let it go. This will eventually break compatibility somewhere and may not be clear to future users of the library that they are not using the same library that node is packaged with. Unless you (and the community as it where) plan to maintain 100% compatibility with Joyent/libuv now and in the future?

In any event I think that a move to make libuv officially a community run project will probably draw more support by the community than if it where to remain a company run project. The way this thread sounds would give a lot of developers the impression that Joyent has just about abandoned the project and may spark a trend of forks that would at the very least be counter-productive to the projects direction.

**Comment by mikeal** [comment]
@mcanthony Joyent has no current contributors working on libuv and hasn't for some time. It has been maintained and released by people outside of Joyent for quite a while. Joyent knows that the community wanted to move the project but declined to comment publicly or to move the project. At the same time, they are not committing to contributing to the project, maintaining or releasing it. The _project_ has moved, it's not a fork, but the benefits of a GH project move will be forgone because Joyent has decided to continue not to participate.
